AISKU/Tests/PerfTests.html

```diff
@@ -5,13 +5,13 @@
     <meta charset="utf-8">
     <meta http-equiv="Cache-control" content="no-Cache" />
     <title>Tests for Application Insights JavaScript AISKU</title>
-    <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-1.23.1.css">
+    <link rel="stylesheet" href="../../../common/Tests/External/qunit-2.9.3.css">
```

Comment:
Most of the changes in this PR are from my investigations to "fix" the GitHub test runs.
This is updating to the current version of GitHub and requireJs -- which did break the CDN Packaging tests because of how they now load and run modules.